components/sceneutil/occlusionquerynode.cpp

```diff
@@ -12,6 +12,9 @@
 #include <osg/ContextData>
 #endif
 
+//ensure minimum popin for in frustum occluded but drastically increase OQ count per frame
+//#define PROVOK_OQ_4_PREVIOUSLY_OCCLUDED 1
```

Comment:
uncomment this to provoke oq for stuff already in frustum but masked previously (increase a lot oq so increase max BVH level)